package.json

```diff
@@ -86,18 +87,18 @@
     "react-test-renderer": "^16.13.1",
     "size-limit": "^4.6.0",
     "tsdx": "^0.14.1",
-    "tslib": "^2.0.1",
+    "tslib": "^2.2.0",
     "typedoc": "^0.20.35",
     "typedoc-plugin-markdown": "^3.6.1",
-    "typescript": "^4.0.3"
+    "typescript": "^4.2.4"
   },
   "dependencies": {
-    "babel-plugin-minify-replace": "^0.5.0",
-    "rxfire": "^4.0.0",
-    "rxjs": "^6.6.3"
+    "rxfire": "5.0.0-rc.0",
+    "rxjs": "^7.0.1"
```

Comment:
We should probably keep 6 while feasible, since 6 => 7 is a big jump for developers.

E.g, `^6.6.3 || ^7.0.1`

We could add to the matrix test like we did RxFire. WDYT?

Reply:
Convo with James: keep 6 in for now to avoid version conflicts if someone is using 6 in their app

Reply:
done